composer.json

```diff
@@ -103,6 +103,7 @@
         "psr-4": {
             "Symfony\\Bridge\\Doctrine\\": "src/Symfony/Bridge/Doctrine/",
             "Symfony\\Bridge\\Monolog\\": "src/Symfony/Bridge/Monolog/",
+            "Symfony\\Bridge\\PhpUnit\\Compat\\": "src/Symfony/Bridge/PhpUnit/Compat/",
```

Comment:
I'm not autoloading the whole bridge but just the Compat classes, otherwise this will not work. Do you have any suggestions for an alternate approach?